### DISCUSSION TITLE
可以增加一个推送间隔的参数吗

### DISCUSSION THREAD

**Discussion by BaeKey** [zdz/ServerStatus-Rust#24] [Q&A] [answered]
我通过网页F12查看到信息是在不停的推送的，希望可以增加一个时间参数，用来设置间隔多少秒推送一次服务器信息，这个我自己判断是在服务端进行的？不是很确定，希望能增加这样一个功能

**Comment by zdz** [comment, marked as answer]
这个不是服务器推送，是网页使用 js 定时拉取的，每隔2秒（2000ms）拉一次服务器数据，你觉得不合适的话可以自己修改

https://github.com/zdz/ServerStatus-Rust/blob/b16b19626141bd9b4dbe6785b4cd5d2a1d30daeb/web/js/serverstatus.js#L316
